src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -409,6 +419,65 @@ const UserGeneralSettings = () => {
                     </div>
                   </>
                 )}
+              {hasPermission(
+                [Permission.AUTO_REQUEST, Permission.AUTO_REQUEST_MOVIE],
+                { type: 'or' }
+              ) && (
+                <div className="form-row">
+                  <label
+                    htmlFor="watchlistSyncMovies"
+                    className="checkbox-label"
+                  >
+                    <span>
+                      {intl.formatMessage(messages.plexwatchlistsyncmovies)}
+                    </span>
+                    <span className="label-tip">
+                      {intl.formatMessage(messages.plexwatchlistsyncmoviestip)}
+                    </span>
+                  </label>
+                  <div className="form-input-area">
+                    <Field
+                      type="checkbox"
+                      id="watchlistSyncMovies"
+                      name="watchlistSyncMovies"
+                      onChange={() => {
+                        setFieldValue(
+                          'watchlistSyncMovies',
+                          !values.watchlistSyncMovies
+                        );
+                      }}
+                    />
+                  </div>
+                </div>
+              )}
+              {hasPermission(
+                [Permission.AUTO_REQUEST, Permission.AUTO_REQUEST_TV],
+                { type: 'or' }
+              ) && (
+                <div className="form-row">
+                  <label htmlFor="watchlistSyncTv" className="checkbox-label">
+                    <span>
+                      {intl.formatMessage(messages.plexwatchlistsyncseries)}
+                    </span>
+                    <span className="label-tip">
+                      {intl.formatMessage(messages.plexwatchlistsyncseriestip)}
+                    </span>
+                  </label>
+                  <div className="form-input-area">
+                    <Field
+                      type="checkbox"
+                      id="watchlistSyncTv"
+                      name="watchlistSyncTv"
+                      onChange={() => {
+                        setFieldValue(
+                          'watchlistSyncTv',
+                          !values.watchlistSyncTv
+                        );
+                      }}
+                    />
+                  </div>
+                </div>
+              )}
```

Comment:
Since this feature is only usable by Plex users, should we hide or fade/disable these options for local users?